tools/pio/pre_custom_esp32.py

```diff
@@ -66,6 +66,7 @@
     "-DUSES_CHART_JS",
 
     "-DUSE_SETTINGS_ARCHIVE",
+    "-DFEATURE_ESPEASY_P2P=1",
```

Comment:
Please test this with either set to 0 or 1, as I'm not sure this is the correct syntax for the PlatformIO build script.